core/src/trezor/ui/loader.py

```diff
@@ -74,13 +74,13 @@ def on_render(self) -> None:
         else:
             s = self.active_style
 
-        Y = const(-24)
```

Comment:
is this change intentional?

Reply:
It's a rename to _Y. It was done by @jpochyla so I left it. You're right that I am not sure what was the rational behind this

Reply:
`const()` without `_` creates a global even in the function scope